cranelift-interpreter/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+//! Cranelift IR interpreter.
+//!
+//! This module is a project for interpreting Cranelift IR.
+
+pub mod clif;
```

Comment:
Miri exposes a lot of things that are only used by [priroda](https://github.com/oli-obk/priroda), which is a graphical debugger based on miri. It may make sense to do the same for `cranelift-interpreter`.

Reply:
Now that `RunCommand` is in `cranelift-reader`, I was able to remove the `clif` module and use that instead. The other modules we probably do want to make public: `interpreter`, `frame`, `environment`.